core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -200,5 +252,54 @@ var ExplorationEditorTranslationTab = function() {
     expect(element(by.css('.protractor-test-translation-feedback-tab'))[0]
     ).toEqual(element(by.css('.oppia-active-translation-tab'))[0]);
   };
+
+  this.openUploadAudioModal = function() {
+    waitFor.elementToBeClickable(
+      uploadAudioButton, 'Upload Audio button is not clickable');
+    uploadAudioButton.click();
+  };
+
+
+  // NOTE: if the state is not visible in the state graph this function will
+  // fail.
+  this.moveToState = function(targetName) {
+    general.scrollToTop();
+    stateNodes.map(function(stateElement) {
+      return stateNodeLabel(stateElement).getText();
+    }).then(function(listOfNames) {
+      var matched = false;
+      for (var i = 0; i < listOfNames.length; i++) {
+        if (listOfNames[i] === targetName) {
+          stateNodes.get(i).click();
```

Comment:
Should we wait for this button to be clickable?

Reply:
I don't think we need to do so because as soon as the translation tab loads fully, the translation graph is ready.